src/query/storages/index/tests/it/filters/bloom_filter.rs

```diff
@@ -63,7 +63,7 @@ fn test_bloom_filter() -> Result<()> {
     ];
     let chunks_ref = chunks.iter().collect::<Vec<_>>();
 
-    let index = BlockFilter::try_create(FunctionContext::default(), schema, &chunks_ref)?;
+    let index = BlockFilter::try_create(FunctionContext::default(), schema, 3, &chunks_ref)?;
```

Comment:
3 --> `V3BloomBlock::VERSION`?

Reply:
I guess this test is expected to test the latest version, not `3`?

Reply:
Yes, 3 is the latest bloom version.

Reply:
I'll update it, there are two struct named BlockFilter

Reply:
So, when we upgrade our version number next time, this test may be missed. Can we define a latest version or another way to archive it?

Reply:
I prefer to add a `LastestBlockBloomFilterIndexVersion = BlockBloomFilterIndexVersion::V3`?

Reply:
BlockBloomFilterIndexVersion::V3 is a enum, we need u64 here.